src/python/dxpy/scripts/dx_build_app.py

```diff
@@ -838,6 +899,8 @@ def _build_app(args, extra_args):
     TODO: remote app builds still return None, but we should fix this.
 
     """
+    _check_suggestions(args.src_dir, args.publish)
```

Comment:
Hi Igor, I ran all the tests in TestDXBuildApp, but these two tests failed:
test_build_applet_with_no_dxapp_json and test_build_applet_with_malformed_dxapp_json
I think the solution is to move the call to _check_suggestions to Line 945 after calling _parse_app_spec, and also to line 756 also after calling _parse_app_spec. Let me know if you think that works. Thanks